includes/class-wc-payments-utils.php

```diff
@@ -177,7 +177,7 @@ public static function interpret_string_exchange_rate(
 	 * @return bool
 	 */
 	public static function is_zero_decimal_currency( string $currency ): bool {
-		if ( in_array( $currency, self::zero_decimal_currencies(), true ) ) {
+		if ( in_array( strtolower( $currency ), self::zero_decimal_currencies(), true ) ) {
```

Comment:
We were comparing `USD` to a list of zero decimal currencies all in a lower format

Reply:
Good point! Could you add a new test for covering this change in `WC_Payments_Utils_Test::test_is_zero_decimal_currency`?

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/7024/commits/0895880cc2d52a84b87cd8d6e928813d57368612